scripts/assert-sorted-checksums.yml

```diff
@@ -40,12 +40,15 @@
     include_vars: ../roles/system_packages/vars/main.yml
 
   - name: Verify that the packages list is sorted
+    loop:
+    - absent
+    - present
```

Comment:
You can use `lookup('vars')` for validation here even if the vars are not under another one.